.github/workflows/slackapp-build-and-test.yml

```diff
@@ -14,10 +14,3 @@ jobs:
       - name: Build the Docker image
         working-directory: "./packages/slack-app"
         run: docker build . --file Dockerfile --tag image
-
-      - name: Run tests
-        working-directory: "./packages/slack-app"
```

Comment:
Does the slack-app no longer have any tests?

Reply:
The slack app still has tests. There needs work to get them running in a pipeline